tests/unit/modin/test_internal_frame.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, sp_df: OrderedDataFrame, internal: InternalFrame) -> None:
         self.internal_frame: InternalFrame = internal
 
 
+@pytest.mark.modin_sp_short_regress
```

Comment:
is it possible to simply mark the whole suite to run as short_regress?

Reply:
Yes, but right now we try to be clear this is only used by modin. Today python API tests are defined in the snowfort code.

Reply:
I think my question is that is it possible to mark the whole suite with one marker instead of marking it one after another? Not about the name, we can keep modin in the name

Reply:
Yes should be possible, we can add this in the next marker PR